components/OpenApplicationsView.tsx

```diff
@@ -1,69 +1,89 @@
-import { Application, GetApplicationDataQuery, TypeformApplication } from "lib/generated/graphql"
-import ApplicationCard from "./typeformApplicationSystem/ApplicationCard";
-import { PopupButton } from "@typeform/embed-react";
-import Button from "./Button";
-import Link from "next/link";
+import { Application, GetApplicationDataQuery, TypeformApplication } from 'lib/generated/graphql';
+import ApplicationCard from './typeformApplicationSystem/ApplicationCard';
+import { PopupButton } from '@typeform/embed-react';
+import Button from './Button';
+import Link from 'next/link';
 
 interface OpenApplicationsViewProps {
-  applications: GetApplicationDataQuery["returnAllOpenApp"];
-  typeformApplications: GetApplicationDataQuery["typeformApplications"];
+  applications: GetApplicationDataQuery['returnAllOpenApp'];
+  typeformApplications: GetApplicationDataQuery['typeformApplications'];
   userData: {
     email: string;
     firstName: string;
     lastName: string;
     major: string;
     netid: string;
     classStanding: string;
-  }
+  };
 }
 
-export default function OpenApplicationsView({ applications, typeformApplications, userData }: OpenApplicationsViewProps) {
-  return <div className="w-full flex flex-col items-center lg:flex-row flex-wrap gap-[30px]">
-    {typeformApplications.map(
-      ({ id, typeformName, description, typeformId, externalResourceUrl, division }) => (
+export default function OpenApplicationsView({
+  applications,
+  typeformApplications,
+  userData,
+}: OpenApplicationsViewProps) {
+  return typeformApplications[0] ? (
```

Comment:
Need to check if both appData and typeformAppData is empty before printing out "No application found".